File: C/URI/1043.c
#include <stdio.h>
 
int main() {
 
  double a, b, c, area, perimetro;
        scanf("%lf %lf %lf", &a, &b, &c);
        if(a < (b + c) && b < (a+c) && c < (a + b))
        {
                perimetro = a + b + c;
                printf("Perimetro = %.1lf\n", perimetro);
        }
        else
        {
                area = (a + b)*c/2;
                printf("Area = %.1lf\n", area);
        }

    return 0;
}

File: php/URI/1043.php
<?php

$linha = explode(' ', readline());

$x = $linha[0];
$y = $linha[1];
$z = $linha[2];

if($x+$y > $z && $x+$z > $y && $z+$y > $x)
{
    echo "Perimetro = ".number_format(($x+$y+$z), 1, ".", "", ). "\n";
}

else
{
    echo "Area = ".number_format((($x+$y)*$z/2), 1, ".", "", ). "\n";
}


?>